### DISCUSSION TITLE
Remove "TAB" text and line from score?

### DISCUSSION THREAD

**Discussion by macca007x** [CoderLine/alphaTab#1696] [Q&A] [answered]
How could I remove the repetitive "TAB" text and line present on every new row?

![image](https://github.com/user-attachments/assets/d2191b75-8b60-4add-aea4-051ed4ea9f0b)

**Comment by Danielku15** [comment]
There are currently no options to hide the [Bracket](https://en.wikipedia.org/wiki/List_of_musical_symbols#Lines) or [Tablature Clef](https://en.wikipedia.org/wiki/List_of_musical_symbols#Clefs)

**Reply by Danielku15** [reply, marked as answer]
I added https://github.com/CoderLine/alphaTab/issues/1697 for the topic of brackets and braces to align better with the usual practices in music notation theory and software. 

With hiding of the tab clef I'm a bit hesitant. The clef is a crucial part in music notation to indicate what notes to expect on a staff.

* Guitar Pro always shows it
* MuseScore shows it. 
* Ultimate Guitar shows it
* Flat.io shows it
* Soundslice shows it

Still: on the other hand for some very simple tab only display needs (e.g. one liners in lectures) it could be useful. Can you share some insights **why** you would like to hide it?
